src/views/home/components/EventCard.tsx

```diff
@@ -67,12 +84,12 @@ const EventCard: FC<EventCardProps> = ({
 
   return (
     <Container>
-      <CoverImage src={coverImage} alt={`cover-${title}`} />
+      <CoverImage src={image} alt={`cover-${title}`} />
       <Box padding="1rem">
         <Label>{category}</Label>
         <Stack spacing="1rem">
           <Title>{title}</Title>
-          <Typography variant="paragraph2">{description}</Typography>
+          <Description>{description}</Description>
           <Stack spacing="0.5rem">
             <Stack direction="row" alignItems="center" spacing="0.5rem">
               <FmdGood sx={{ color: '#8c8c8c', width: '1.125rem' }} />
```

Comment:
```suggestion
              <FmdGood sx={{ color: 'neutral400.main', width: '1.125rem' }} />
```
![image](https://github.com/PurwadhikaDev/purwadhika-corporate-website/assets/61060248/aee8985a-35e3-455d-b81e-928b9a9a3351)